.all-contributorsrc

```diff
@@ -4,6 +4,7 @@
   ],
   "imageSize": 100,
   "commit": false,
+  "badgeTemplate": "[ac-badge]: https://img.shields.io/badge/all_contributors-<%= contributors.length %>-orange.svg?style=flat",
```

Comment:
Nice, looks like this works! Just the one last note, everything else is color 14cc21 and not `style=flat`, so let's go with that?

```suggestion
  "badgeTemplate": "[ac-badge]: https://img.shields.io/badge/all_contributors-<%= contributors.length %>-14cc21.svg",
```